tests/tacacs/utils.py

```diff
@@ -51,8 +51,16 @@ def setup_tacacs_client(duthost, creds_all_duts, tacacs_server_ip):
 
     # enable tacacs+
     duthost.shell("sudo config aaa authentication login tacacs+")
-    duthost.shell("sudo config aaa authorization local")
-    duthost.shell("sudo config aaa accounting disable")
+
+    try:
+        duthost.shell("sudo config aaa authorization local")
+    except RunAnsibleModuleFail:
+        logger.info("config aaa authorization command not support on this version")
```

Comment:
Suggest add DUT version number to the log information.

Reply:
Thanks, I will submit another PR for this.
Please revert the UTs to unblock others.